### ISSUE TITLE
Variables used but not declared

### ISSUE THREAD

**Issue by thatsvenki038** [JunweiLiang/social-distancing-prediction#14]
config.traj_class_loss_weight, config.grid_loss_weight, config.act_loss_weight and config.wd in line 694, 705,751and 765 are used without initializing them in models.py?

Can u address this issue?  This issue is also present in google/next prediction.  What are initial values for these variables?

**Comment by JunweiLiang** [comment]
This code is only for inference so the loss weights are not used. It should run just fine. In the next-prediction repo, they are initialized in the [training script](https://github.com/google/next-prediction/blob/master/code/train.py#L109).
